src/elements/content-sidebar/activity-feed/common/activity-timestamp/ActivityTimestamp.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../../../styles/variables';
 
 .bcs-ActivityTimestamp {
-    color: $bdl-gray-80;
+    color: $bdl-gray;
```

Comment:
Can we put the color under as a value ender a subclass. ActivityTimestamp is used in a lot of places and we don't want to subtly change them right now.
 
.bcs-Comment-separator{
.  bcs-ActivityTimestamp {
    color: $bdl-gray-80 !important;	

}